linuxdeploy-plugin-gtk.sh

```diff
@@ -237,5 +237,16 @@ done
 
 # set write permission on lib64 again to make it deletable.
 chmod +w "$APPDIR"/usr/lib64 || true
+
+# We have to copy the files first to not get permission errors when we assign gio_extras_dir
```

Comment:
are you some kind of linux guru? :joy:

Reply:
i wish 😂  i spend more or less the whole day to find that stuff. There are like no resources at all other than some unresolved github issues with somewhat similar problems lol. Trial and error for the win i guess 😪